services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/volumes_resolver.py

```diff
@@ -2,6 +2,66 @@
 from pathlib import Path
 from typing import Any, Dict
 
+from settings_library.rclone import RCloneSettings, S3Provider
+from models_library.projects_nodes_io import NodeID
+from models_library.projects import ProjectID
+
+from .errors import DynamicSidecarError
+
+
+def _get_s3_volume_driver_config(
+    r_clone_settings: RCloneSettings,
+    project_id: ProjectID,
+    node_uuid: NodeID,
+    volume_name: str,
+) -> Dict[str, Any]:
+    driver_config = {
+        "Name": "rclone",
+        "Options": {
+            "type": "s3",
+            "s3-access_key_id": r_clone_settings.S3_ACCESS_KEY,
+            "s3-secret_access_key": r_clone_settings.S3_SECRET_KEY,
+            "s3-endpoint": r_clone_settings.endpoint_url,
+            "path": f"{r_clone_settings.S3_BUCKET_NAME}/{project_id}/{node_uuid}/{volume_name}",
```

Comment:
ok so do I get it right that the data will end up in
e.g. ```BUCKET/PROJECT_ID/NODE_ID/VOLUME_PATH_IN_CONTAINER``` ??

Reply:
I've replied above with an example. But yes, that's the point of it. I will change the volume_name parameter in this function as it is not that meaningful.